docs/source/utils.rst

```diff
@@ -2,6 +2,12 @@
 utils
 *****
 
+engineering_database.py
+--------------------
```

Comment:
@Johannes-Sahlmann A bit of a gotcha, but `sphinx` will freak out if the length of the underline doesn't match what is above it.

Reply:
Done.